### ISSUE TITLE
⚡️ Weekly meetings ⚡️

### ISSUE THREAD

**Issue by mkg20001** [mercode-org/meros#1]
Here you'll find the planned and held merOS meetings

**Comment by mkg20001** [comment]
## MerOS Weekly Meeting

#### When? DD-MM-YYYY 17 UTC / 19 DE / 20 Turkey
#### Where? Online

- Jitsi Link: Will be announced on t.me/merosdistro at 16:45 UTC
- Notes: https://hackmd.io/yD5YfxPxQ7u2fvBa2TIhrg

**Comment by mkg20001** [comment]
## MerOS Weekly Meeting

#### When? DD-MM-YYYY 16 UTC / 18 DE / 19 Turkey
#### Where? Online

- Jitsi Link: Will be announced on t.me/merosdistro at 15:45 UTC
- Notes: https://hackmd.io/2chmOQWJTpe0PRrr11KXwg?both

**Comment by Yutyo** [comment]
@mkg20001 Yes guys, we will have a meeting tomorrow at this date. Is it OK for everyone?
